codec/test_codec.go

```diff
@@ -86,6 +88,12 @@ type MyInnerStruct3 struct {
 	F   Foo           `serialize:"true"`
 }
 
+type MyStructWithPtr struct {
+	N1 *int32   `serialize:"true,nullable"`
+	N2 *int64   `serialize:"true,nullable"`
+	N3 []*int32 `serialize:"true,nullable"`
```

Comment:
Can we add tests for nullable arrays + maps as well?

Reply:
@StephenButtolph is 168627d95 good enough?